CONTRIBUTING.md

```diff
@@ -152,3 +152,54 @@ Getting help with git
 ---------------------
 
 If you're new to ``git``, you'll probably find [gitref.org](http://gitref.org/) helpful.
+
+Setting up a development environment
+------------------------------------
+
+The recommended way to set up a development environment for contributing to scikit-bio is using [Anaconda](https://store.continuum.io/cshop/anaconda/) by Continuum Analytics, with it's associated command line utility `conda`. The primary benefit of `conda` over `pip` is that on some operating systems (ie Linux), `pip` installs packages from source. This can take a very long time to install Numpy, scipy, matplotlib, etc. `conda` installs these packages using pre-built binaries, so the installation is much faster. Another benefit of `conda` is that it provides both package management and virtual environments, which removes the necessity of using `virtualenv` separately. Not all packages are available using `conda`, therefore our strategy is to install as many packages as possible using `conda`, then install any remaining packages using `pip`.
+
+The following steps can be used to manually set up a `conda` development environment with all prerequisites for developing scikit-bio. Alternatively, you can simply run the `create_dev_env` script from the scikit-bio directory:
+
+```
+./create_dev_env env_name
+```
+
+##### 1. Install Anaconda
+See [Continuum's site](https://store.continuum.io/cshop/anaconda/) for instructions.
+
+##### 2. Create a new virtual environment
+```
+conda create -y -n env_name python=3.4 pip
```

Comment:
Interesting, thanks Jai! I just tested creating a new 2.7 and 3.5 environments (just released!) without including pip and both ended up installing pip. I guess it makes sense for conda to bundle pip in their environments (particularly for 2.7) because otherwise it could create havoc.